apps/cowswap-frontend/src/modules/hooksStore/utils/fetchWithTimeout.ts

```diff
@@ -0,0 +1,23 @@
+import { getTimeoutAbortController } from '@cowprotocol/common-utils'
```

Comment:
I was suggesting to move it to `@cowprotocol/common-utils`, you might want to use this in other front-ends and not just CoW Swap. Anyways, we can always move when needed in other apps if this is adding some overhead

Reply:
@anxolin I see now it was added to the common-utils but didn't cleanup the older util function. Just pushed a fix for this now.